src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,29 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.12.0
+29 August 2023
+
+This version is compatible with Kubernetes versions 1.24 to 1.26.
+
+### Breaking changes
+
+- Kubeconfigs downloaded from the Okteto web UI now require Okteto CLI 2.15 or higher to use dynamic tokens for authenticating against Kubernetes. <!-- 6972 -->
+
+### Features
+
+- The Okteto CLI has been upgraded to version [2.19.0](https://github.com/okteto/okteto/releases/tag/2.19.0).
+- The Admin view has been redesigned with a new vertical tab column. <!-- 6851 -->
+- A new section has been added to the Admin view for listing "Registry Credentials". (link to docs TBC) <!-- 6851 -->
```

Comment:
This PR is adding the section in the docs to be linked here: https://github.com/okteto/docs/pull/480.

@omnomagonz Could you help us to write a small reflect about our reasoning behind this e.g. how we believe it’s more usable, and that it will help us as we increase the admin-level features

Reply:
```suggestion
- Added "Registry Credentials" to the Admin view to display configured private registry information. This provides visibility into the private registries in use for your Okteto instance (username/password and AWS Access/Secret only for now) (link to docs TBC) <!-- 6851 -->
```

Would this work?

Reply:
Resolving for the sake of keeping this moving. Feel free to modify further :)